reference/runners/docker.rst

```diff
@@ -0,0 +1,717 @@
+    └──────────────────────────────────────────────────┘
+
+    Dockerfile path: '</my/runner/forder>/Dockerfile'
+    Docker build context: '</my/runner/forder>'
+
+    Step 1/5 : ARG BASE_IMAGE
+
+    Step 2/5 : FROM $BASE_IMAGE
+
+    ...
+
+    Successfully built 286df085400f
+    Successfully tagged my-conan-runner-image:latest
+
+    ...
+
+    ┌───────────────────────────────┐
+    | Creating the docker container |
+    └───────────────────────────────┘
+
+
+    ┌─────────────────────────────────────────┐
+    | Container my-conan-runner-image running |
+    └─────────────────────────────────────────┘
+
+
+    ┌─────────────────────────────────────────┐
+    | Running in container: "conan --version" |
+    └─────────────────────────────────────────┘
+
+    ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+    | Restore host cache from: </my/runner/forder>/conan-center-index/recipes/zlib/all/.conanrunner/docker_cache_save.tgz |
+    └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+
+    Restore: zlib/1.3.1 in p/zlib95420566fc0dd
+    Restore: zlib/1.3.1:b647c43bfefae3f830561ca202b6cfd935b56205 in p/zlibd59462fc4358e/p
+    Restore: zlib/1.3.1:b647c43bfefae3f830561ca202b6cfd935b56205 metadata in p/zlibd59462fc4358e/d/metadata
+
+    ┌────────────────────┐
+    | Stopping container |
+    └────────────────────┘
+
+If we now check the status of our conan and docker cache, we will see the zlib package compile for Linux and the new docker image and container.
+
+.. code-block:: bash
+
+    $ conan list "*:*"
+    Found 1 pkg/version recipes matching * in local cache
+    Local Cache
+    zlib
+        zlib/1.3.1
+        revisions
+            e20364c96c45455608a72543f3a53133 (2024-04-29 17:18:07 UTC)
+            packages
+                b647c43bfefae3f830561ca202b6cfd935b56205
+                info
+                    settings
+                    arch: x86_64
+                    build_type: Release
+                    compiler: gcc
+                    compiler.version: 11
+                    os: Linux
+                    options
+                    fPIC: True
+                    shared: False
+
+    $ docker ps --all
+    CONTAINER ID   IMAGE                   COMMAND                  CREATED          STATUS                       PORTS     NAMES
+    1379072ae424   my-conan-runner-image   "/bin/bash -c 'while…"   17 seconds ago   Exited (137) 2 seconds ago             my-conan-runner-image
+
+    $ docker images  
+    REPOSITORY        TAG       IMAGE ID       CREATED          SIZE
+    my-conan-runner   latest    383b905f352e   22 minutes ago   531MB
+    ubuntu            22.04     437ec753bef3   12 days ago      77.9MB
+
+If we run the conan create command again we will see how conan reuses the previous container beacause we set ``remove=False``.
```

Comment:
```suggestion
If we run the ``conan create`` command again we will see how Conan reuses the previous container because we have set ``remove=False``.
```